.github/workflows/run-tests-with-coverage.yml

```diff
@@ -36,25 +36,22 @@ jobs:
           - 5432:5432
 
     steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-
-      - name: Set up PHP
```

Comment:
let's revert the `name` param, it's good to see the human readable name of the job